src/components/CurriculumComponents/CurriculumHeader/CurriculumHeader.tsx

```diff
@@ -86,10 +88,10 @@ const CurriculumHeader: FC<CurriculumHeaderPageProps> = ({
       scroll: false,
     },
     {
-      label: "Overview",
-      page: "curriculum-overview",
+      label: cycleTwoEnabled ? "Explainer" : "Overview",
```

Comment:
@assadk88 I think we should only go with this change, and not the URL changes.

In theory I like the changing the URL, however we'll have to think about redirects and SEO effects (see comment in notion ticket)

I suggest closing this PR, so we can attach a link to a backlog ticket for when we do change the URLs. Then opening a new PR with just the label changes